src/features/landing/components/ForWhoSection/Cards/Cards.stories.tsx

```
@@ -0,0 +1,20 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
```

ESLint

[error] `@storybook/react` type import should occur before import of `react`

(import/order)